jenkins-jobs/pipelines/build-debezium-images.groovy

```diff
@@ -181,6 +182,6 @@ node('Slave') {
             }
         }
     } finally {
-        mail to: 'jpechane@redhat.com', subject: "${JOB_NAME} run #${BUILD_NUMBER} finished", body: "Run ${BUILD_URL} finished with result: ${currentBuild.currentResult}"
+        mail to: ${MAIL_ID}, subject: "${JOB_NAME} run #${BUILD_NUMBER} finished", body: "Run ${BUILD_URL} finished with result: ${currentBuild.currentResult}"
```

Comment:
@ani-sha In all cases you should either use MAIL_ID or `"${MAIL_ID}"` as you don't define it as string now but referes to it as variable in string.